maps-compose-utils/src/main/java/com/google/maps/android/compose/kml/data/KmlStyleId.kt

```diff
@@ -0,0 +1,17 @@
+package com.google.maps.android.compose.kml.data
+
+public abstract class KmlStyleId {
+    private var styleId: String = ""
+
+    internal fun getId(): String = styleId
+
+    /**
+     * Sets id, making sure there is consistently a single # as prefix
+     *
+     * @return id with # as prefix
+     */
+    internal fun setId(id: String): String {
+        styleId = if (id.startsWith("#")) id else "#${id}"
```

Comment:
vraagje, waarom return je hier id ipv styleId

Reply:
Goeie, is ook helemaal niet meer nodig om terug te geven